scipy/stats/tests/test_fit.py

```diff
@@ -136,4 +136,3 @@ def test_expon_fit():
     data = [0, 0, 0, 0, 2, 2, 2, 2]
     phat = stats.expon.fit(data, floc=0)
     assert_allclose(phat, [0, 1.0], atol=1e-3)
```

Comment:
```suggestion
    assert_allclose(phat, [0, 1.0], atol=1e-3)

```
Per PEP8, end files with newline character, so let's not remove this.

Reply:
Unless you use "Commit suggestion", please leave these unresolved so I can check. Thanks!

Reply:
Interesting. I think what happened is that github closed this comment automatically after i committed https://github.com/scipy/scipy/pull/13298/commits/1cf2e367f781ee5c2d78994d4f20a03548566191 . (see message). However there are no diffs in that commit. Very weird.

Reply:
Ok that is _very_ weird. Committed again from "commit suggesttion" and did not captured the diff https://github.com/scipy/scipy/pull/13298/commits/d5e3022f83a5aa87bc91858c41b76e0441243d56 .Possibly github does not like the diff to be only the whitespace at the end of the file... Will try and fix that locally

Reply:
Huh. OK. Looks fine locally.